Zotero/Models/FeatureGates.swift

```diff
@@ -42,6 +44,10 @@ struct FeatureGates: OptionSet {
 #if FEATURE_GATE_DOWNLOAD_FILES_AT_SYNC
         gates.insert(.downloadFilesAtSync)
 #endif
+
+#if FEATURE_GATE_SPEECH
+        gates.insert(.speech)
```

Comment:
Speech requires PDF worker as well.
```suggestion
        gates.insert(.speech)
        gates.insert(.pdfWorker)
```